parm/convection_permitting/namelist.init_atmosphere

```diff
@@ -44,6 +44,7 @@
 &preproc_stages
     config_static_interp = false
     config_native_gwd_static = false
+    config_native_gwd_gsl_static = false
```

Comment:
Do we need to set up lines 46 and 47 on the fly?

Reply:
If we don't use gsl_static, why do we bother to add this namelist option?

Reply:
The default value is true but it might be only used when generating static.nc
So for ic and lbc tasks, it is better to set it to false.